tests/ut/scargo_check/ut_copyright_checker.py

```diff
@@ -106,15 +107,21 @@ def test_check_copyright_fix(
     [(FILE_CONTENTS_WITH_INCORRECT_COPYRIGHT,)],
     indirect=True,
 )
-def test_check_copyright_no_fix_incorrect(
-    mock_file_contents: MagicMock, config: Config, mock_find_files: MagicMock
+def test_check_copyright_fix_2(
```

Comment:
This is an awful name for a test. It would be best to parametrize `test_check_copyright_fix` so that both cases are covered by that function.

Reply:
It is strange because at my site theres no that kind of code abominations.

Reply:
Solved according to solution proposal.